scripts/installation/install_solc.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import platform
+import shutil
+from os.path import dirname, join, abspath
+from os import stat, chmod
+import requests
+
+
+if platform.system() != 'Linux':
```

Comment:
What is preventing support of other platforms - a lack of available binary?

Reply:
Github also has link for windows, but that is zip archive which has exe and few dll libraries, so need aditional operations. For me much easier to maintain this manually. But if we need it, I can add this for windows and I think using information from #977 - for mac